smart-contracts/test/withdrawals/op.ethereum.ts

```diff
@@ -84,7 +84,7 @@ describe('op withdrawal helper', function () {
     })
   })
 
-  it.skip('should work for the base sequence using an ERC20', async () => {
```

Comment:
This one is from  (Mar-11-2025 02:12:33 PM +UTC)

Reply:
ok that one is passing now

Reply:
woot!